MonoGame.Framework/Media/MediaPlayer.WMS.cs

```diff
@@ -260,11 +275,16 @@ private static void OnSessionStopped()
             if (_nextSong != null)
             {
                 if (_nextSong != _currentSong)
-                    StartNewSong(_nextSong);
+                    StartNewSong(_nextSong, _nextSongStartPosition);
                 else
-                    StartSession(PositionBeginning);
+                    StartSession(_nextSongStartPosition.HasValue ? PositionVariantFor(_nextSongStartPosition.Value) : PositionBeginning);
                 _nextSong = null;
             }
         }
+
+        private static Variant PositionVariantFor(TimeSpan position)
+        {
+            return new Variant { Value = (long)(10000000L * (position.TotalSeconds)) };
```

Comment:
The `Value` calculation can be simplified to: `return new Variant { Value = position.Ticks };`
And you can simplify things even further by moving the null check into `PositionVariantFor` instead. e.g.:
```c#
private static Variant PositionVariantFor(TimeSpan? position)
{
    return position.HasValue ? new Variant { Value = position.Value.Ticks } : PositionBeginning
}
```
That way you don't need to do this check each time you call this method.